CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+1.11.0 (beta)
```

Comment:
sorry, I did typo on last comment on this thread (wanted to say that next will be `1.11.1`
My comment was not about version itself (`1.11.0` is completely okay) but about this `beta` addition. We usually don't have _betas_, because this package not in the beta stage. It was released already.